src/router.js

```diff
@@ -23,7 +24,11 @@ const mustBeAuthenticated = (to, from, next) => {
   if (store.state.auth.authenticated) {
     return next();
   }
-  return next('/login');
+  store.dispatch('getReturningUser');
+  if (store.state.auth.isReturningUser) {
```

Comment:
localstorage can only store strings so I'd pref that this be a more explicit check for 'true'. Also since the method returns the value I think it may be nicer to capture it in a variable here
```suggestion
  const isReturningUser = store.dispatch('getReturningUser');
  if (isReturningUser === 'true') {
```